styles.css

```diff
@@ -0,0 +1,97 @@
+/* Melanie Cascante */
+
+h1 {
+  width: 220px;
+  height: 100px;
+  display: flex;
+  padding-top: 50px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+/* ic_menu */
+
+.ic_menu {
+  padding-top: 15px;
+  padding-left: 60px;
+  background: #fafafa;
+}
+
+/* greeting */
+
+.greeting {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  background: #fff;
+  flex: 1;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  padding-top: 0;
+  padding-right: 150px;
+  padding-bottom: 20px;
+  color: #1a2236;
+}
```

Comment:
Hey 👋🏽 last thing but not least, make sure your design is centered and the padding is the same on both sides 👀  See below 👇🏽 

## Your design
![Screenshot 2023-01-14 at 12 13 35](https://user-images.githubusercontent.com/45739963/212486030-ce12f8ad-d5db-40a1-85be-d0ed77a213ee.png)


## Figma design
![Screenshot 2023-01-14 at 12 13 21](https://user-images.githubusercontent.com/45739963/212485971-ce102b5a-5157-4a65-8b73-b51c67fb23ce.png)

Let me know if you have any questions ❓ and keep going with this project. You rock 💪🏽